test/VocabularyTest.cpp

```diff
@@ -116,17 +116,26 @@ TEST(VocabularyTest, IncompleteLiterals) {
 }
 
 TEST(Vocabulary, PrefixFilter) {
-  RdfsVocabulary voc;
-  voc.setLocale("en", "US", true);
-  ad_utility::HashSet<string> s;
+  RdfsVocabulary vocab;
+  vocab.setLocale("en", "US", true);
+  ad_utility::HashSet<string> words;
 
-  s.insert("\"exa\"");
-  s.insert("\"exp\"");
-  s.insert("\"ext\"");
-  s.insert(R"("["Ex-vivo" renal artery revascularization]"@en)");
-  voc.createFromSet(s);
+  words.insert("\"exa\"");
+  words.insert("\"exp\"");
+  words.insert("\"ext\"");
+  words.insert(R"("["Ex-vivo" renal artery revascularization]"@en)");
+  vocab.createFromSet(words);
```

Comment:
Maybe it is not too hard to implement a createFromSet for testing that also takes for each word the bool on whether it should be externalized, but I currently don't know how hard it is to manually create an external vocab for testing.